pkg/drivers/kic/oci/oci.go

```diff
@@ -142,6 +142,13 @@ func CreateContainerNode(p CreateParams) error {
 		"--label", p.NodeLabel,
 	}
 
+	// network
+	if p.OCIBinary == Docker && runtime.GOOS == "linux" { // for now only docker on linux
```

Comment:
why only on linux?

Reply:
I spoke to Jose, and he said:

Docker containers on OSx don’t support ip! It is always localhost
There is why we have the custom tunnel for example